source/core_http_client.c

```diff
@@ -1953,58 +1962,95 @@ static HTTPStatus_t receiveAndParseHttpResponse( const TransportInterface_t * pT
 {
     HTTPStatus_t returnStatus = HTTPSuccess;
     size_t totalReceived = 0U;
-    size_t currentReceived = 0U;
+    int32_t currentReceived = 0U;
```

Comment:
```suggestion
    int32_t currentReceived = 0;
```

Reply:
Should this be changed to `bytesReceived` or have the other one changed to `currentSent` for consistency?

Reply:
Thank you